### PR TITLE
Make docker image

### LINKED ISSUE #19: Make a Docker image
Suggested by @Pierstoval [on Twitter](https://twitter.com/Pierstoval/status/1192406525383454720): create an official Docker image.

### PR DESCRIPTION
**Pull request by nicfb**
| Q              | A
| -------------- | ---
| Bug fix?       | no
| New feature?   | yes
| Related issues | Fix #19 
| Has BC-break   | no
| License        | GNU AGPL-v3

**Checklist:**
- [ x] I have updated the manpages

This PR adds a Docker image for Kosmorro. It will be added to Docker Hub on push to master.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+USER kosmorro
+
+WORKDIR /home/kosmorro
+
+COPY kosmorrolib/ kosmorrolib/
+
+COPY kosmorro .
+
+RUN python kosmorro
```
Comment (Deuchnord) [inline Dockerfile:17]: Can you remove this call?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -81,3 +81,31 @@ jobs:
               run: |
                 make POEDITOR_API_ACCESS="${POEDITOR_API_ACCESS}" POEDITOR_PROJECT_ID="306433" build
                 twine upload dist/*
+    docker:
+      name: Push to DockerHub
+      runs-on: ubuntu-latest
+      steps:
+        -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```
Comment (Deuchnord) [inline .github/workflows/release.yml:90]: Is this really needed?

**Review by Deuchnord — CHANGES_REQUESTED** [review]
Thanks for your PR!

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+certifi==2020.12.5
```
[the remark this answers is not in the document]
Comment (nicfb) [reply requirements.txt]: I have added a commit that removes this file.

File: README.md [hunk 4]
```diff
@@ -32,6 +32,12 @@ Now, you can install Kosmorro with your PIP: `pip3 install kosmorro`.
 
 Kosmorro being at an early-stage development, Windows is not supported officially for now.
 
+#### Docker
+
+Get the official Kosmorro Docker image by running `docker pull kosmorro/kosmorro`.
+Now that you have the image, you can run it with `docker run -it kosmorro`.
+Finally, run Kosmorro by executing `python kosmorro` in the container.
```
[the remark this answers is not in the document]
Comment (nicfb) [reply README.md]: I have added this to the Dockerfile using the ENV instruction.

[the remark this answers is not in the document]
Comment (nicfb) [reply README.md]: I have updated the README.

File: Dockerfile (hunk 1 above)
Comment (nicfb) [reply Dockerfile]: Done!

File: .github/workflows/release.yml (hunk 2 above)
Comment (nicfb) [reply .github/workflows/release.yml:90]: I'm not entirely sure, to be honest. I was going off of what was at the top of the page: "v2 of this action includes significant updates and now uses Docker Buildx. It works with 3 new actions (login, setup-buildx and setup-qemu) that we have created".

Comment (Deuchnord) [reply .github/workflows/release.yml:90]: Oh, I see, OK for me, then!

File: Dockerfile [hunk 5]
```diff
@@ -2,18 +2,24 @@ FROM python:3.9-slim
...
+WORKDIR /home/kosmorro
 
-RUN pip install -r requirements.txt
+ENV PATH="/home/kosmorro:${PATH}"
 
-USER kosmorro
+RUN python -m pip install --upgrade pip
 
-WORKDIR /home/kosmorro
+RUN pip install pipenv
```
[the remark this answers is not in the document]
Comment (nicfb) [reply Dockerfile:11]: Oh yeah, thanks! I just added that.

**Comment by Deuchnord** [comment]
Thanks!
I'm doing some tests on my side and looking at what's wrong with the tests in #139, and I'll merge this this week 🙂

**Comment by nicfb** [comment]
Awesome, thanks for your help!

**Comment by Deuchnord** [comment]
I've just merged the fixes for the CI.
Can you rebase your branch on `master`, squash your commits and change its message to comply with the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/), please? :slightly_smiling_face: 

If you need help, feel free to ask!

**Comment by nicfb** [comment]
Sure! I'm thinking the type I would use here would be "docs". Is that right?

**Comment by Deuchnord** [comment]
You can use the `feat` type, since to me, it is a new feature :)

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (nicfb) [reply README.md]: Is this resolved now?

**Comment by Deuchnord** [comment]
Thank you for your PR, I'll look at it this weekend :)

**Review by Deuchnord — APPROVED** [review]

**Comment by Deuchnord** [comment]
Thank you for your work, @nicfb! :rocket:

**Pull request merged**